collectd.env

```diff
@@ -0,0 +1,23 @@
+# if monitoring only one MySQL server
```

Comment:
It is file holding environment variables not config. See here: https://docs.docker.com/compose/compose-file/ (env_file)